### PR TITLE
Fixes #43: Adds fork version to navbar title, links to Github tree

### LINKED ISSUE #43: Display the Airflow fork version with hyperlink to this repo
The git hash of Airflow itself would be in `env.var["AIRFLOW_GIT_HASH"]` which would point to something like this:
https://github.com/coatue/airflow/commit/82257aa04de8283ac7e97c31243418930cc3637c

image

### PR DESCRIPTION
**Pull request by gmarks-coatue**
The environment variable `AIRFLOW_GIT_HASH` should contain the short git hash of Airflow. The link points to `https://github.com/coatue/airflow/tree/<git-hash>`.

Before:
Screen Shot 2019-07-31 at 4 39 46 PM

After:
Screen Shot 2019-07-31 at 4 39 06 PM

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -104,6 +104,9 @@
     # filter_by_owner if authentication is enabled and filter_by_owner is true
     FILTER_BY_OWNER = not current_app.config['LOGIN_DISABLED']
 
+@current_app.context_processor
+def navbar_vars():
```
Comment (pathikrit) [inline airflow/www/views.py]: I would just make this `return os.environ.get['AIRFLOW_GIT_HASH']` and move the logic to construct the URL (or handle the case if it is None in the view

File: airflow/www/views.py [hunk 2]
```diff
@@ -105,6 +105,11 @@
     FILTER_BY_OWNER = not current_app.config['LOGIN_DISABLED']
 
 
+@current_app.context_processor
+def navbar_vars():
+    return {'git_hash': os.environ['AIRFLOW_GIT_HASH']}
```
Comment (pathikrit) [inline airflow/www/views.py]: this would crash if git hash is missing right? I am fine with that as long as the cause is evident ... cc: @awong-coatue

Comment (gmarks-coatue) [reply airflow/www/views.py]: Yeah, it would; should I go ahead w/ merge or wait?

Comment (pathikrit) [reply airflow/www/views.py]: yeah merge all approved PRs

**Pull request merged**